chainerrl/experiments/evaluator.py

```diff
@@ -344,6 +351,14 @@ def evaluate_and_update_max_score(self, t, episodes):
             self.max_score = mean
             if self.save_best_so_far_agent:
                 save_agent(self.agent, "best", self.outdir, self.logger)
+                if not os.path.exists(os.path.join(self.outdir, 'best')):
+                    try:
+                        os.makedirs(os.path.join(self.outdir, 'best'))
+                    except OSError as err:
+                        if err.errno != errno.EEXIST:
+                            raise
+                record_best_eval(os.path.join(self.outdir, 'best'),
+                                 t, self.max_score)
```

Comment:
- `best` directory corresponds to an internal state of the agent state, so it is not intended to contain other files.
- Why do we need to save `best/best.txt`? We can see the max evaluation score in `scores.txt`.